src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -587,7 +592,40 @@ bool clang_c_convertert::get_function(const clang::FunctionDecl &fd, exprt &)
   return false;
 }
 
+bool clang_c_convertert::get_function_body(
+  const clang::FunctionDecl &fd,
+  exprt &new_expr)
+{
+  assert(fd.hasBody());
+
+  exprt body_exprt;
+  if(get_expr(*fd.getBody(), body_exprt))
+    return true;
+
+  new_expr = body_exprt;
+  return false;
+}
+
 bool clang_c_convertert::get_function_params(
+  const clang::FunctionDecl &fd,
+  code_typet::argumentst &params)
+{
+  if(!fd.parameters().size()) // return if no parameter
+    return false;
+
+  for(auto const &pdecl : fd.parameters())
+  {
+    code_typet::argumentt param;
+    if(get_function_param(*pdecl, param))
```

Comment:
In which case do we return true?

Reply:
Done.
Added a comment to explain a failed case.